opus/application/static_media/js/widgets.js

```diff
@@ -51,6 +51,9 @@ var o_widgets = {
         $("#op-search-widgets").sortable({
             items: "> li",
             cursor: "move",
+            // Specify the elements that will not trigger sorting when being grabbed. This is to make
```

Comment:
Debby has a pull that prevents drag on anything except the widget title. I think that eliminates the need for this change.